README.md

```diff
@@ -49,6 +48,31 @@ An example setup using this plugin can be found at [turtlebot3-UE](https://githu
 - some basic numerical types are not natively supported in Blueprint (e.g. double, unsigned int). In order to use these, a workaround is needed (a plugin implementing those types for BP, a modified UE4 or a custom implementation).
 - In autogenerated messages, the method MsgToString() should be implemented by the user as its current purpose is to help debugging.
 
+# How to update ROS inside RclUE
+Currently there is a Scripts/update_ros.py for automatic build and update ROS2 (foxy) libraries. To update ROS:
+- Install 'ros2' (foxy) (until section 'Build the code in the workspace') [Building ROS 2 on Ubuntu Linux](https://docs.ros.org/en/foxy/Installation/Ubuntu-Development-Setup.html)
+- Reinstall python package due to [issue](https://github.com/ros-visualization/qt_gui_core/issues/212):
+```
+sudo apt remove shiboken2 libshiboken2-dev libshiboken2-py3-5.14
+pip3 install --user shiboken2
+```
+- Apply patch for ros2 to avoid LD_LIBRARY_PATH dependency issue:
+```
+src/ros2/rcpputils/src/find_library.cpp:
+in function find_library_path():
+---//return "";
++++return "lib" + library_name + ".so";
+```
+- Clone ['rclc' package](https://github.com/ros2/rclc/tree/foxy), branch 'foxy' (it's not a part of official ros2)
+- Clone ['ue_msgs'](https://github.com/rapyuta-robotics/UE_msgs) (Rapyuta-Robotics project) 
+- Run Scripts/update_ros.py script with arguments:
+```
+python3 update_ros.py <Path to UE4> <Path to ROS2 foxy> <Path to rclc package> <Path to UE_msgs> <Path to project, which includes RclUE plugin>
```

Comment:
Could you also note the specific destination folder of those `paths to`, probably some example would be helpful to make it clear.

Reply:
I could see an example in `update_ros.py`:
`python3 update_ros.py /home/vilkun/UE/UnrealEngine /home/vilkun/ros2_foxy /home/vilkun/work/build_foxy/rclc /home/vilkun/work/build_foxy/UE_msgs /home/vilkun/work/turtlebot3-UE`
Could you also specify here in README also?